### DISCUSSION TITLE
I will change my gpu how keep current setting  ?

### DISCUSSION THREAD

**Discussion by alex3xelax** [Rem0o/FanControl.Releases#2850] [Q&A]
I currently have the fan control software set up to manage the case, CPU, and GPU fans.

In a few days, I will be changing the graphics card. How can I **keep my current configuration/setting**s and only update the sensors for the new hardware, in this case, the graphics card I will be installing?

Thank you

**Comment by Rem0o** [comment]
IF you update for the same type ( nvidia for nvidia ), not much to do, but swap in the sensors for the new card, then calibrate/setup the fans for that new GPU.

**Reply by alex3xelax** [reply]
yes, i update nvidia for nvidia. So when i start the program with the new gpu(2 fans model as old gpu) i just set start/stop basicaly or i need redetect the gpu fans sensor  ?

![Immagine 2024-11-25 233244](https://github.com/user-attachments/assets/5a90efbe-6013-4fe7-a7ba-b2727a940c99)

**Reply by Rem0o** [reply]
Yes. You can rerun the automatic detection from the 3 dot menu on each card. Give that a save, and should then be good to go.

**Reply by alex3xelax** [reply]
ok thanks, i report when i have installed the new gpu.

**Reply by alex3xelax** [reply]
hi,
after some retry  automatic detection plus override zero fans control as reported [here ](https://github.com/Rem0o/FanControl.Releases/discussions/1372) it's works fine with rtx 4070 super evo :)
I can control the zero fans speed. ^^
